modules/apps/portal/portal-verify-extender/src/main/java/com/liferay/portal/verify/extender/internal/osgi/commands/VerifyProcessTrackerOSGiCommands.java

```diff
@@ -384,6 +393,19 @@ private boolean _isInitialDeployment(
 		return false;
 	}
 
+	private boolean _isServiceBundle(Bundle bundle) {
```

Comment:
@shuyangzhou 

can we use this method instead?
https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/module/util/BundleUtil.java#L42

Or at least merge both in one?

Reply:
They are doing different checking logic, and BundleUtil one has other usages, which most likely will fail stuff if you force to use the logic here. I would not touch isLiferayServiceBundle for this change.